app/grants/urls.py

```diff
@@ -20,7 +20,7 @@
 from django.urls import path, re_path
 
 from grants.views import (
-    add_grant_from_collection, bulk_fund, bulk_grants_for_cart, cancel_grant_v1, cart_thumbnail, clr_grants,
+    add_grant_from_collection, bulk_fund, bulk_grants_for_cart, cancel_grant_v1, cart_thumbnail, clr_grants, collage,
```

Comment:
what is collage? I don't see it here

Reply:
It's here -- https://github.com/gitcoinco/web/pull/8674